src/OpenApiLinter.php

```diff
@@ -154,11 +159,13 @@ private function processOutput($path, $output_category, $severity) {
     $messages = array();
     foreach ($output_category as $output) {
       $message = new ArcanistLintMessage();
+      $data = $output[0];
```

Comment:
According to the [sample output above](https://github.com/pinterest/arcanist-linters/blob/227bfee6483b998eb09fb7ff45eeb7ba22abbbb5/src/OpenApiLinter.php#L116-L135), this will not work.

Is the sample output wrong? Did the openapi validator output format change in a more recent version (more likely)?

If the latter, you should update the version requirement in the README so that users of this library know what the new minimum version is like as done in a8dcb7a74dd1be971bccbc62ef220108fa994af0.